anchor/common/ssv_types/src/share.rs

```diff
@@ -0,0 +1,51 @@
+use crate::CommitteeID;
+use std::time::SystemTime;
+use types::{Address, Domain, Graffiti, PublicKey};
+
+type ValidatorIndex = usize;
+
+/// Share of a key that a operator owns and accompanying metadata
+#[derive(Debug, Clone)]
+pub struct SSVShare {
+    pub share: Share,
+    pub metadata: Metadata,
```

Comment:
You've been awesome with comments on all the fields, might as well be consistent here